xml/System.Linq/Enumerable.xml

```diff
@@ -8257,18 +8257,12 @@ In Visual Basic query expression syntax, an `Aggregate Into Max()` clause transl
         <param name="keySelector">A function to extract the key for each element.</param>
         <summary>Returns the maximum value in a generic sequence according to a specified key selector function.</summary>
         <returns>The value with the maximum key in the sequence.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-If `TKey` is a reference type and the source sequence is empty or contains only values that are `null`, this method returns `null`.
-
-          ]]></format>
-        </remarks>
+        <remarks>If <typeparamref name="TSource" /> is a reference type and the source sequence is empty or contains only values that are <see langword="null" />, this method returns <see langword="null" />.</remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="source" /> is <see langword="null" />.</exception>
         <exception cref="T:System.ArgumentException">No key extracted from <paramref name="source" /> implements the <see cref="T:System.IComparable" /> or <see cref="T:System.IComparable`1" /> interface.</exception>
+        <exception cref="T:System.InvalidOperationException">
```

Comment:
I think this should be included in the remarks section as well. @gewarren thoughts?

Reply:
Yes, good idea.